src/journal/test/JournalTest.hs

```diff
@@ -143,9 +141,7 @@ appendBSFake' bs fj@(FakeJournal rles ix termCount) =
     noBackPressure =
       if journalLength' > termLen * termCount
       then if journalLength < termLen * termCount
-           then [(FakeJournal (if sum (map fst padding) == 0
-                             then rles
-                             else Vector.snoc rles padding) ix (termCount + 1), Left Rotation)]
+           then [(FakeJournal (Vector.snoc rles padding) ix (termCount + 1), Left Rotation)]
```

Comment:
can't comment where I want, but in `appendBSFake'` we should update `readBytes` as well.